xml/signals/CTC-MB/index.shtml

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
```

Comment:
This file needs to be the same (including name) as the index.php files used in other signaling systems so it can be rendered on the JMRI website correctly.

Reply:
I certainly hosed that file up.  Corrections made.